thrift/src/main/thrift/atoms/qanda.thrift

```diff
@@ -0,0 +1,18 @@
+namespace * contentatom.qanda
+namespace java com.gu.contentatom.thrift.atom.qanda
+#@namespace scala com.gu.contentatom.thrift.atom.qanda
+
+include "../shared.thrift"
+
+struct QAndAAtom {
+  1: optional string typeLabel
+  2: optional list<string> tags
+  3: optional Image eventImage
+  4: required list<QAndAItem> items
+  5: optional string sidenote
+}
+
+struct QAndAItem {
+  1: optional string title
+  2: required string body
```

Comment:
hmm, that is distracting. Here is the mockup:
![snippets_q_and_a2](https://cloud.githubusercontent.com/assets/629976/26624038/9294bd78-45e7-11e7-9cf2-aba7e4898c2a.png)

The title of the atom will be the question that is answered in the explainer. We don't know whether the content of the answer will be a single `body` block or a collection, and whether the `title` will ever be useful here... just wanted to leave that door open. I guess we can add it later if needed?